connectors/Egnyte/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "capabilities": [
+      "actions",
+      "triggers"
+    ],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "xx877eb8mujv6dyu23w5v899",

```

Comment:
Please remove the client id and replace with a dummy value. If this client id is used in your production please discard it and use a new one.